src/features/applet-calendar/ui/AppletCalendar.tsx

```diff
@@ -0,0 +1,67 @@
+import { FC } from 'react';
+
+import { CalendarProvider, ExpandableCalendar } from 'react-native-calendars';
+
+import { format } from '@shared/lib/utils';
+import { Text, YStack, Box } from '@shared/ui';
+
+const AppletCalendar: FC = () => {
+  let currentDate = new Date();
+  const weekDay = format(currentDate, 'i');
+  const weekDayNumeric = +weekDay + 1;
+
+  let firstDay = weekDayNumeric > 7 ? weekDayNumeric - 7 : weekDayNumeric;
```

Comment:
how it is possible to get weekDayNumeric > 8?
this code-row looks like workaround